content/en/opentelemetry/otel_collector_datadog_exporter.md

```diff
@@ -662,12 +662,36 @@ The OpenTelemetry Collector has [two primary deployment methods][20]: Agent and
 
 Datadog provides out-of-the-box dashboards that you can copy and customize. To use Datadog's out-of-the-box OpenTelemetry dashboards:
 
-1. Install the [OpenTelemetry integration][23].
+1. Install the [OpenTelemetry integration][24].
 2. Go to **Dashboards** > **Dashboards list** and search for `opentelemetry`:
 
    {{< img src="metrics/otel/dashboard.png" alt="The Dashboards list, showing two OpenTelemetry out-of-the-box dashboards: Host Metrics and Collector Metrics." style="width:80%;">}}
 
-The **Host Metrics** dashboard is for data collected from the [host metrics receiver][21]. The **Collector Metrics** dashboard is for any other types of metrics collected, depending on which [metrics receiver][22] you choose to enable.
+The **Host Metrics** dashboard is for data collected from the [host metrics receiver][25]. The **Collector Metrics** dashboard is for any other types of metrics collected, depending on which [metrics receiver][26] you choose to enable.
+
+
+### Containers overview dashboard
+
+The [Docker Stats][27] receiver generates container metrics for the OpenTelemetry Collector. The Datadog Exporter translates container metrics to their Datadog counterparts.
+
+Use the following configuration to enable additional attributes on the Docker Stats receiver that populates the containers overview dashboard:
+
+```yaml
+  docker_stats:
+    metrics:
+      container.network.io.usage.rx_packets:
+        enabled: true
+      container.network.io.usage.tx_packets:
+        enabled: true
+      container.cpu.usage.system:
+        enabled: true
+      container.memory.rss:
+        enabled: true
+      container.blockio.io_serviced_recursive:
+        enabled: true
```

Comment:
Can we update this to the config we are running in demo account ? There are two important  metrics `container.uptime` and `container.memory.hierarchical_memory_limit` missing

Reply:
Done! Thanks for checking so thoroughly.